scripts/test.sh

```diff
@@ -13,3 +13,9 @@ npm run eslint -- -c config/node.js config/node.js "$@"
 
 # react config will fail against its self, so test against any empty file
 printf "" | npm run eslint -- --stdin -c config/react.js "$@"
+
+npm run check-deprecated --deprecated config/babel.js
+npm run check-deprecated --deprecated config/chai.js
+npm run check-deprecated --deprecated config/console.js
+npm run check-deprecated --deprecated config/mocha.js
+npm run check-deprecated --deprecated config/react.js
```

Comment:
No need for the flag here, since it's added in the package.json.

```suggestion
npm run check-deprecated config/babel.js
npm run check-deprecated config/chai.js
npm run check-deprecated config/console.js
npm run check-deprecated config/mocha.js
npm run check-deprecated config/react.js
```